core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -530,5 +542,62 @@ public void accept(IracResponseCollector.Result result, Throwable throwable) {
       synchronized boolean isCompleted() {
          return stateStatus == StateStatus.COMPLETED;
       }
-   }
+     }
```

Comment:
can you check the identation? we use 3 spaces (no tabs :))

Reply:
You can use the setting files in the `ide-settings` dir to correctly setup the project.

Reply:
Push the changes